content/joomla/joomdev-astroid.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Astroid Framework"
+details: https://astroidframework.com/
+demo: https://demo.joomdev.com/astroid-framework?utm_source=astroid_framework_home
```

Comment:
Should be `https://demo.joomdev.com/astroid-framework/toolbar` which is the URL to the actual template. The URL provided links to a demo site with the template inside an iframe

Reply:
I have changed the URL and committed new templates as well. 

Please check if now it is good or not.